### DISCUSSION TITLE
Idea stealers

### DISCUSSION THREAD

**Discussion by b1tw0rker** [poweradmin/poweradmin#491] [General]
This project take your ideas and does not credit you.

**Comment by edmondas** [comment]
> This project take your ideas and does not credit you.

Would you please clarify what exactly went wrong? Which idea was stolen without identifying the contributors? Although I am not sure if all the ideas are original, sometimes what matters is the implementation returned for everyone's benefit.

When I release a version, I always credit the people who contributed, if I don't, it's either because there were too many changes, or because I forgot.

**Comment by j4Hu** [comment]
idea is nothing without execution :)

**Comment by edmondas** [comment]
@b1tw0rker Okay, I reviewed all the pull requests. So there was only one PR from you that added bootstrap and dark theme. The idea of adding bootstrap is not new, check PR #275 also https://github.com/poweradmin/poweradmin/issues/168#issuecomment-56353829

In my opinion, your PR wasn't clean enough, it had commented code, used fontawesome (why not bootstrap icons?) and so on. In my role as a project maintainer, I often spend time cleaning up things implemented poorly by previous pull requests, so I decided to close this PR since another implementation was done more cleanly.

**Comment by edmondas** [comment]
@b1tw0rker Your contribution has been noted in the 3.1.0 release.
https://github.com/poweradmin/poweradmin/releases/tag/v3.1.0

**Comment by edmondas** [comment]
I've updated the contributing guidelines for PowerAdmin at https://github.com/poweradmin/poweradmin?tab=readme-ov-file#contributing. Please take a moment to review these changes.

To better understand my position and values behind these updates, I've also written a detailed explanation at https://www.poweradmin.org/p/open-source-reality-a-maintainers.

These guidelines explain what I expect from contributors and how I manage the project. Thanks for your contributions.
